projects/innov2017/generate_stack.py

```diff
@@ -52,7 +52,9 @@ def add_service(self, container):
                 'WORDPRESS_DB_HOST': self.db_container_name,
                 'WORDPRESS_DB_PASSWORD': db_password,
                 'WORDPRESS_DB_NAME': db_name,
-                'VIRTUAL_HOST': virtual_hosts
+                'VIRTUAL_HOST': virtual_hosts,
+                'LETSENCRYPT_HOST': virtual_hosts,
+                'LETSENCRYPT_EMAIL': 'ssl-contact@beia.ro',
```

Comment:
Let's not hardcode that here. You can choose a default to be used as default if it is not specified  by the JSON, but let's not hardcode.

Reply:
Which one do you mean? the email?

Reply:
⚡

Reply:
Both of them.